testapps/testapp/.settings/org.eclipse.buildship.core.prefs

```diff
@@ -0,0 +1,2 @@
+connection.project.dir=../../..
```

Comment:
Do you know what this file is?

Reply:
I think this needs to be added to git ignore.  Not sure where it came from.